CorsixTH/Src/th_gfx.h

```diff
@@ -484,129 +489,129 @@ class THAnimationManager
+    bool hit_test_morph(int iDestX, int iDestY, int iTestX, int iTestY);
+    void draw_child(render_target* pCanvas, int iDestX, int iDestY);
+    bool hit_test_child(int iDestX, int iDestY, int iTestX, int iTestY);
+
+    THLinkList* get_previous() {return m_pPrev;}
+    size_t get_animation() const {return animation_index;}
+    bool get_marker(int* pX, int* pY);
+    bool get_secondary_marker(int* pX, int* pY);
+    size_t get_frame() const {return frame_index;}
+    int get_crop_column() const {return crop_column;}
+
+    void set_animation(animation_manager* pManager, size_t iAnimation);
+    void set_morph_target(animation *pMorphTarget, unsigned int iDurationFactor = 1);
+    void set_frame(size_t iFrame);
+
+    void set_speed(int iX, int iY) {speed.x = iX, speed.y = iY;}
+    void set_crop_column(int iColumn) {crop_column = iColumn;}
 
     void persist(LuaPersistWriter *pWriter) const;
     void depersist(LuaPersistReader *pReader);
 
-    THAnimationManager* getAnimationManager(){ return m_pManager;}
+    animation_manager* get_animation_manager(){ return manager;}
 private:
-    THAnimationManager *m_pManager;
-    THAnimation* m_pMorphTarget;
-    size_t m_iAnimation; ///< Animation number.
-    size_t m_iFrame;     ///< Frame number.
+    animation_manager *manager;
+    animation* morph_target;
+    size_t animation_index; ///< Animation number.
+    size_t frame_index;     ///< Frame number.
     union {
         struct {
-            //! Amount to change m_iX per tick
+            //! Amount to change x per tick
             int x;
-            //! Amount to change m_iY per tick
+            //! Amount to change y per tick
             int y;
         } speed;
         //! Some animations are tied to the marker of another animation and
         //! hence have a parent rather than a speed.
-        THAnimation* m_pParent;
+        animation* parent;
     };
 
-    size_t m_iSoundToPlay;
-    int m_iCropColumn;
+    size_t sound_to_play;
+    int crop_column;
 };
 
-class THSpriteRenderList : public THAnimationBase
+class sprite_render_list : public animation_base
 {
 public:
-    THSpriteRenderList();
-    ~THSpriteRenderList();
+    sprite_render_list();
+    ~sprite_render_list();
 
     void tick();
-    void draw(THRenderTarget* pCanvas, int iDestX, int iDestY);
-    bool hitTest(int iDestX, int iDestY, int iTestX, int iTestY);
+    void draw(render_target* pCanvas, int iDestX, int iDestY);
+    bool hit_test(int iDestX, int iDestY, int iTestX, int iTestY);
 
-    void setSheet(THSpriteSheet* pSheet) {m_pSpriteSheet = pSheet;}
-    void setSpeed(int iX, int iY) {m_iSpeedX = iX, m_iSpeedY = iY;}
-    void setLifetime(int iLifetime);
-    void appendSprite(size_t iSprite, int iX, int iY);
-    bool isDead() const {return m_iLifetime == 0;}
+    void set_sheet(sprite_sheet* pSheet) {sheet = pSheet;}
+    void set_speed(int iX, int iY) {speed_x = iX, speed_y = iY;}
```

Comment:
a comma?! :p

Reply:
Esoteric C operator for no reason... another on for the list.